index.md

```diff
@@ -5,43 +5,43 @@ layout: base
 # Features
 ### Section description
 
+Dune is a build system for OCaml projects.
+Using it, you can build executables, libraries, run tests, and much more.
+
 <div class="flex-grid">
 <div class="col text-container">
-#### Great thing
-Dune does amazing things that will change your life. You will
-wonder what you did without it and thank the OCaml wizard every
-day.
+#### Dune knows OCaml and its ecosystem
+Dune has rules that precisely capture how the different compilers work.
+It interoperates with most of the tools in the OCaml toolchain like OPAM,
+merlin, reason, and `js_of_ocaml`.
 
-#### Better thing
-Dune does amazing things that will change your life. You will
-wonder what you did without it and thank the OCaml wizard every
-day.
+#### Dune is fast
+Dune works hard to do things once and in parallel, so both cold and incremental
+builds are way faster than traditional build systems.
 
-#### Most wonderful thing
-Dune does amazing things that will change your life. You will
-wonder what you did without it and thank the OCaml wizard every
-day.
+#### Dune is widely used
+Dune is used in both large projects and small libraries.
+About 30% of OPAM packages are built using Dune.
```

Comment:
Maybe we should give examples of big projects using dune, such as lwt or coq

Reply:
coq doesn't seem to be using dune. maybe we can include mirage?

Reply:
or just leave that out and we'll be put that in "featured uses" below.

Reply:
There is a PR to switch coq to dune, I believe coq developers are waiting for the 1.0.0, but yh we should wait until it's merged. Putting it in featured uses seems good